pubtools/pulplib/_impl/fake/match.py

```diff
@@ -131,10 +132,26 @@ def match_in(matcher, field, obj):
 
 @visit(LessThanMatcher)
 def match_field_less(matcher, field, obj):
+    if isinstance(matcher._value, six.string_types):
```

Comment:
This now make behavior inconsistent if someone uses matcher that goes via match_field_eq(), can you apply the same there?